backends/smt2/Makefile.inc

```diff
@@ -3,14 +3,30 @@ OBJS += backends/smt2/smt2.o
 
 ifneq ($(CONFIG),mxe)
 ifneq ($(CONFIG),emcc)
+
+# MSYS targets support yosys-smtbmc, but require a launcher script
+ifeq ($(CONFIG),$(filter $(CONFIG),msys2 msys2-64))
```

Comment:
There is no `msys2-64` config in the Yosys build system.

Reply:
Actually there is ( https://github.com/YosysHQ/yosys/blob/master/Makefile#L245 ) he added it time ago

Reply:
Actually there is ( https://github.com/YosysHQ/yosys/blob/master/Makefile#L245 ) he added it time ago

Reply:
weird... oh! my Yosys win32 working copy is on an old branch. Sorry, my fault.